ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml

```diff
@@ -64,6 +64,9 @@ spec:
 #              - mount_point: /var/imageregistry
 #                size: 102500
 #                start: 344844
+       # allocate partitions for persist storage used by HTTP transport subscription data for PTP and BMER operators.
+       # disk id and and size needs to be adjusted to the hardware
+       # ignitionConfigOverride: '{"ignition":{"version":"3.2.0"},"storage":{"disks":[{"device":"/dev/disk/by-id/wwn-0x11111000000asd123","wipeTable":false,"partitions":[{"sizeMiB":16,"label":"httpevent1","startMiB":350000},{"sizeMiB":16,"label":"httpevent2","startMiB":350016}]}],"filesystem":[{"device":"/dev/disk/by-partlabel/httpevent1","format":"xfs","wipeFilesystem":true},{"device":"/dev/disk/by-partlabel/httpevent2","format":"xfs","wipeFilesystem":true}]}}'
```

Comment:
Can these partitions be done using the same `diskPartition: ` structure used above? Is there a preference/reason to use the ignitionConfigOverride instead?

Reply:
A few reasons:
1. diskPartition field only works with 1 partition even though it looks to be a list field
2. rootfs has to be used via diskPartition (I think I read it somewhere, never tried to use another disk though) 
3. not sure how to add label to partition using diskPartition
4. we also need to create filesystem using those partitions, so we can use LSO localvolume CR directly.  
Basically ignition configs seems to be the best option that provides everything we needed with minimal configs.